2019Labs/CustomSecurityContent/documentation/lab3_profiles.adoc

```diff
@@ -521,36 +525,64 @@ selections:
 .. When you have finished editing,
 press `Ctrl+X`, then enter `y` to save and exit.
 
-. Use templates to generate checks automatically.
+
+=== 3.6.2 Use templates to generate checks automatically
+
+You have successfully defined the rule and added it to the profile.
+However, the rule currently has no check nor remediation.
+That means that Openscap can't check if the Hexchat package is installed.
+You have now two options. You can either write a new check and respective remediations, or you can use an already created template which will generate respective checks and remediations for you.
```

Comment:
This is not always an option - templates exist only in cases of rules performing common tasks. I would mention that querying the template documentation page for keywords may be a good idea.